Plugins/Flow.Launcher.Plugin.PluginsManager/PluginsManager.cs

```diff
@@ -120,7 +120,10 @@ internal async Task InstallOrUpdate(UserPlugin plugin)
                             .ChangeQuery(
                                 $"{Context.CurrentPluginMetadata.ActionKeywords.FirstOrDefault()} {Settings.HotkeyUpdate} {plugin.Name}");
 
-                    Application.Current.MainWindow.Show();
+                    var mainWindow = Application.Current.MainWindow;
+                    mainWindow.Visibility = Visibility.Visible;
+                    mainWindow.Focus();
+
```

Comment:
Will the mainwindow hide at these time? Haven't taken a try but I think we may not need to change the visibility

Reply:
yeah it will hide because of the message prompt